apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -56,9 +61,28 @@ function StakeForm({
         setFieldValue('amount', maxToken);
     }, [maxToken, setFieldValue]);
 
+    const calculateRemaining = useMemo(() => {
+        if (!maxToken) return 0;
+        return roundFloat(+maxToken - +amount, DECIMAL_PLACE);
```

Comment:
We should do this calculation with mist/not dominated values and use `useFormatCoin` to do the final format to display it

Reply:
Got it